waflyctl.go

```diff
@@ -25,7 +25,7 @@ import (
 	"time"
 
 	"github.com/BurntSushi/toml"
-	"github.com/sethvargo/go-fastly/fastly"
+	"github.com/fastly/go-fastly/fastly"
```

Comment:
this is really unrelated. please move it to a different commit or a separate pr.